cmd/protoc-gen-gorums/dev/zorums_ordered_gorums.pb.go

```diff
@@ -529,48 +529,51 @@ func (s *GorumsServer) RegisterOrderingUnaryRPCHandler(handler OrderingUnaryRPCH
 // OrderingFuture asynchronously invokes a quorum call on configuration c
 // and returns a FutureResponse, which can be used to inspect the quorum call
 // reply and error when available.
-func (c *Configuration) OrderingFuture(ctx context.Context, in *Request) (*FutureResponse, error) {
+func (c *Configuration) OrderingFuture(ctx context.Context, in *Request) *FutureResponse {
 	fut := &FutureResponse{
 		NodeIDs: make([]uint32, 0, c.n),
 		c:       make(chan struct{}, 1),
 	}
-	id, expected, replyChan, err := c.orderingFutureSend(ctx, in)
-	if err != nil {
-		return nil, err
-	}
-	go func() {
-		defer close(fut.c)
-		c.orderingFutureRecv(ctx, in, id, expected, replyChan, fut)
-	}()
-	return fut, nil
-}
-
-func (c *Configuration) orderingFutureSend(ctx context.Context, in *Request) (uint64, int, chan *orderingResult, error) { // get the ID which will be used to return the correct responses for a request
+	// get the ID which will be used to return the correct responses for a request
 	msgID := c.mgr.nextMsgID()
 
 	// set up a channel to collect replies
 	replyChan := make(chan *orderingResult, c.n)
 	c.mgr.putChan(msgID, replyChan)
 
+	expected := c.n
```

Comment:
We can probably avoid the `expected` variable and just use `c.n` or `c.Size()`; at least I don't see any changes to expected in the code.

Reply:
It only changes if `perNodeArg` is enabled.